src/@next/pages/CheckoutPage/CheckoutPage.tsx

```diff
@@ -190,8 +200,15 @@ const CheckoutPage: React.FC<IProps> = ({}: IProps) => {
         break;
     }
   };
-  const handleStepSubmitSuccess = (data?: object) => {
-    if (activeStepIndex === steps.length - 1) {
+  const handleStepSubmitSuccess = (
+    currentStep: CheckoutStep,
+    data?: object
+  ) => {
+    const activeStepIndex = getActiveStepIndex();
+    if (
+      currentStep === CheckoutStep.Review ||
+      location.pathname === "/checkout/payment-confirm"
```

Comment:
Why do we have to check the path here? Isnt Checkout step depending on url already?

Reply:
Yes, it is. Removed